src/Files.App/Views/MainPage.xaml

```diff
@@ -267,17 +266,8 @@
 						Grid.Column="0"
 						HorizontalAlignment="Stretch"
 						HorizontalContentAlignment="Stretch"
-						Background="{ThemeResource App.Theme.FileArea.BackgroundBrush}"
-						BackgroundSizing="InnerBorderEdge"
-						BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
-						BorderThickness="1"
 						Content="{x:Bind ((viewmodels:MainPageViewModel)DataContext).SelectedTabItem.ContentFrame, Mode=OneWay}"
-						CornerRadius="8"
-						Translation="0,0,8">
-						<ContentPresenter.Shadow>
-							<ThemeShadow x:Name="PageContentThemeShadow" />
-						</ContentPresenter.Shadow>
-					</ContentPresenter>
+						CornerRadius="8" />
```

Comment:
Adding a padding of `4` fixes the shadow issue, so my theory is that the shadow is there and it's getting clipped.